README.rst

```diff
@@ -23,6 +23,27 @@ https://gradle.org/install/
 install docker
 https://docs.docker.com/install/
 
+JDK 8
+-----
+
+The version of gradle we're using is old and doesn't work on JDK 11. For Ubuntu
+18.04 do the following to downgrade to JDK 8.
+
+.. code:: bash
+
+    sudo apt install openjdk-8-jdk
+    update-java-alternatives --set java-1.8.0-openjdk
+
+To confirm you have JDK 8 run the following and make sure you get simlar output
+with a version that starts with 1.8.
+
+.. code:: bash
+
+    $ java -version
+    openjdk version "1.8.0_222"
+    OpenJDK Runtime Environment (build 1.8.0_222-8u222-b10-1ubuntu1~18.04.1-b10)
+    OpenJDK 64-Bit Server VM (build 25.222-b10, mixed mode) update-java-alternatives --set java-1.8.0-openjdk
+
```

Comment:
Might be helpful to call out the fact that it syncs up (+/-) to what we use in production here: https://github.com/edx/configuration/blob/master/playbooks/roles/oraclejdk/defaults/main.yml#L5. What do you think?